### PR TITLE
feat(Answer:1): projection

### PR DESCRIPTION
**Pull request by svenson95**
## Checklist for challenge submission

- [x] Start your PR message with Answer:${challenge_number}

### CODE REVIEW

File: apps/angular/projection/src/app/ui/card/card.component.ts [hunk 1]
```diff
@@ -1,61 +1,47 @@
...
-          [name]="item.firstName"
-          [id]="item.id"
-          [type]="type"></app-list-item>
-      </section>
+    <section>
+      @for (item of list; track item.id) {
+        <ng-template
+          [ngTemplateOutlet]="listItemTemplate"
+          [ngTemplateOutletContext]="{ $implicit: item }"></ng-template>
+      }
```
Comment (tomalaforge) [inline apps/angular/projection/src/app/ui/card/card.component.ts:23]: 👍

File: apps/angular/projection/src/app/ui/card/card.component.ts [hunk 2]
```diff
@@ -1,61 +1,47 @@
...
-
-  addNewItem() {
-    if (this.type === CardType.TEACHER) {
-      this.teacherStore.addOne(randTeacher());
-    } else if (this.type === CardType.STUDENT) {
-      this.studentStore.addOne(randStudent());
-    }
-  }
+  @ContentChild(ListItemRefDirective, { read: TemplateRef })
+  listItemTemplate!: TemplateRef<{ $implicit: T }>;
```
Comment (tomalaforge) [inline apps/angular/projection/src/app/ui/card/card.component.ts:46]: 👍

**Comment by tomalaforge** [comment]
THat's perfect. I have nothing to add. amazing job

**Comment by svenson95** [comment]
thx for the review 👍

File: apps/angular/projection/src/app/ui/card/card.component.ts (hunk 1 above)
Comment (webbomj) [reply apps/angular/projection/src/app/ui/card/card.component.ts:23]: Why here used ng-template with ngTemplateOutlet? In docs used ng-container with ngTemplateOutlet

**Pull request closed without merging**